src/view/Aktivitetskort.tsx

```diff
@@ -34,7 +34,7 @@ export function Aktivitetskort(props: Props) {
             );
         }
     }
-    return null;
+    return <span className="aktivitetkort" />;
```

Comment:
Hvorfor returnere en tom span ?

Reply:
For de dialogene som ikke har en aktivitet ønsker jeg at det finnes et like stor "boks" der. Men mye mulig man burde gjøre dette annerledes